test/distributed_exec.jl

```diff
@@ -1272,6 +1272,16 @@ if DoFullTest
     rmprocs(pids)
 end
 
+# Test addprocs/rmprocs from master node only
+for f in [ ()->addprocs(1), ()->rmprocs(workers()) ]
+    try
+        remotecall_fetch(f, id_other)
```

Comment:
this needs an `error("unexpected")` otherwise it'll silently do nothing if it succeeds